### PR TITLE
Add introduction post

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

**Comment by lcarva** [comment]
We could probably spend some time into making the post page a little bit nicer. It looks like a wall of text even though there are quite a few code blocks.

**Comment by simonbaird** [comment]
I want to add some extra CSS for code blocks, let me have a try.

**Comment by simonbaird** [comment]
https://github.com/enterprise-contract/enterprise-contract.github.io/pull/49 looks better to me at least.

File: website/content/posts/introducing-the-enterprise-contract.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+---
+
+You may have heard of [sigstore](https://www.sigstore.dev/how-it-works) and its container image
+verification tool, [cosign](https://docs.sigstore.dev/cosign/overview/). This blog post introduces a
+policy-driven workflow, [Enterprise Contract](https://enterprisecontract.dev/ec/main/index.html),
+built on those technologies.
+
+## Before starting…
+
+Let’s say you have an image, quay.io/lucarval/demo:ec, and you want to verify this image was signed
```
Comment (simonbaird) [inline website/content/posts/introducing-the-enterprise-contract.md]:
```suggestion
Let’s say you have an image, `quay.io/lucarval/demo:ec`, and you want to verify this image was signed
```

File: website/content/posts/introducing-the-enterprise-contract.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+format. From its success, we can tell that the image signature and the image attestations match the
+provided public key. We can also see some metadata information about the found attestations.
+
+Let’s take a step further and say we want to validate the SLSA Provenance attestations with a
+certain set of policy rules. The [ec-policies](https://github.com/enterprise-contract/ec-policies)
+repo has a set of useful rego policies, so let’s use that.
+
+Create a policy file called `policy.yaml` with the following content:
+
+```text
```
Comment (simonbaird) [inline website/content/posts/introducing-the-enterprise-contract.md]: If we do this, does it highlight?

```suggestion
```yaml
```

Comment (simonbaird) [reply website/content/posts/introducing-the-enterprise-contract.md]: I will try it..

Comment (lcarva) [reply website/content/posts/introducing-the-enterprise-contract.md]: I think we should it regardless. Maybe style it better later if we want. I also broke down the last call to the ec-cli into two blocks, one with the command (text) and one with the output (yaml). That looks a bit better.

**Review by simonbaird — APPROVED** [review]

**Pull request merged**